pkg/sql/execinfrapb/processors_bulk_io.proto

```diff
@@ -480,6 +480,11 @@ message CloudStorageTestSpec {
   // NEXT ID: 3;
 }
 
+message TableReplicationMetadata {
```

Comment:
also, we'll need to store the destination table name -- we can't rely on the source table name being the same as the destination name sadly.